<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-AM810LT" name="Test" tests="2" failures="1" timestamp="2 Jan 2020 16:32:17 GMT" time="27.972" errors="0">
  <testcase name="@AfterMethod takePhoto" time="0.0" classname="testcases.LoginTestActiTime">
    <failure type="org.apache.commons.mail.EmailException" message="Invalid message supplied">
      <![CDATA[org.apache.commons.mail.EmailException: Invalid message supplied
	at org.apache.commons.mail.SimpleEmail.setMsg(SimpleEmail.java:42)
	at generic.UtilityClass.sendEmail(UtilityClass.java:147)
	at generic.BaseClass.takePhoto(BaseClass.java:68)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod takePhoto -->
  <testcase name="@BeforeMethod pageLoad" time="0.0" classname="testcases.LoginTestActiTime">
    <skipped/>
  </testcase> <!-- @BeforeMethod pageLoad -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="testcases.LoginTestActiTime">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="invalidLoginApp" time="8.705" classname="testcases.LoginTestActiTime">
    <failure type="java.lang.AssertionError" message="expected [actiTIME - Login] but found [actiTIME]">
      <![CDATA[java.lang.AssertionError: expected [actiTIME - Login] but found [actiTIME]
	at testcases.LoginTestActiTime.invalidLoginApp(LoginTestActiTime.java:53)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- invalidLoginApp -->
  <testcase name="validLoginApp" time="0.026" classname="testcases.LoginTestActiTime">
    <skipped/>
  </testcase> <!-- validLoginApp -->
</testsuite> <!-- Test -->
